django-stubs/db/backends/base/operations.pyi

```diff
@@ -8,12 +8,8 @@ from django.db.backends.utils import CursorWrapper
 from django.db.models.base import Model
 from django.db.models.expressions import Case, Expression
 from django.db.models.sql.compiler import SQLCompiler
-
-from django.db import DefaultConnectionProxy
 from django.db.models.fields import Field
 
-_Connection = Union[DefaultConnectionProxy, BaseDatabaseWrapper]
```

Comment:
Here dropped the distinction between `DefaultConnectionProxy` and `BaseDatabaseWrapper`.